src/app/page.tsx

```diff
@@ -255,8 +255,11 @@ async function getPageData(spoofDate?: string) {
 
       const active = drops.filter((drop) => {
         const comparison = compareAsc(now, drop.endDate)
+        const isExternal = !!drop.externalLink && !drop.buttonText
+        const isInfo =
+          (drop.address === '0x0' || drop.address === '0x') && !isExternal
```

Comment:
@lauchness remind me your question here?

Reply:
Let's pause on this altogether. Chatting with Aneri and I think we need more control than this (e.g. ability to prioritize certain drops, etc.) - sorry for the rework!